pkg/harvester/dialog/HarvesterMaintenanceDialog.vue

```diff
@@ -97,11 +97,17 @@ export default {
           label-key="harvester.host.enableMaintenance.force"
         />
       </div>
-      <Banner color="warning" :label="t('harvester.host.enableMaintenance.protip')" class="mb-0" />
-      <Banner v-for="(err, i) in errors" :key="i" color="error" :label="err" />
 
-      <div v-if="unhealthyVM">
-        <Banner color="error mb-5">
+      <Banner color="warning" :label="t('harvester.host.enableMaintenance.protip')" />
+      
+      <Banner v-for="(err, i) in errors" :key="i" color="error" :label="err"  />
```

Comment:
It would be better to uppcase the error message first character.

```
<Banner v-for="(err, i) in errors" :key="i" color="error" :label="ucFirst(error)" />
```

Reply:
Sorry, I means the L115 
```
{{ ucFirst(unhealthyVM.message) }}
```

<img width="554" alt="image" src="https://github.com/user-attachments/assets/c244c2b4-e58e-44eb-b5d7-b398793a8672">